### PR TITLE
 1.0.0 release

### LINKED ISSUE #510: [decouple] Updater 1.0.0
https://github.com/owncloud/updater/releases/tag/v1.0.0RC1

to be tested as part of 10.2 RC1 QA

@patrickjahns FYI

### PR DESCRIPTION
**Pull request by patrickjahns**
closes #510

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## [1.1.0]
```
Comment (VicDeo) [inline CHANGELOG.md]: @patrickjahns 1.1.0?

Comment (patrickjahns) [reply CHANGELOG.md]: thanks - didn't spot this before - seems to be a typo by @PVince81 
Will fix

Comment (patrickjahns) [reply CHANGELOG.md]: 🤔 - would a bad changelog trigger a 1.0.1 release?

Comment (PVince81) [reply CHANGELOG.md]: probably not worth it... but can keep it in the pipe of change for whatever the next release will be...

**Pull request merged**